### PR TITLE
docs: Glob Import Asの翻訳の変更

### PR DESCRIPTION
**Pull request by sapphi-red**
https://github.com/vitejs/docs-ja/pull/474#issuecomment-1158950848 の件の変更です

### CODE REVIEW

File: guide/features.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ const modules = {
 }
 ```
 
-### glob インポート
+### glob インポートでの形式の変換
```
Comment (sapphi-red) [inline guide/features.md:312]: 原文

> ### Glob Import As

File: guide/features.md [hunk 2]
```diff
@@ -327,7 +327,7 @@ const modules = {
 }
 ```
 
-`{ as: 'url' }` はアセットの URL としての読み込みもサポートしています。
+`{ as: 'url' }` によるアセットの URL としての読み込みもサポートしています。
```
Comment (sapphi-red) [inline guide/features.md:330]: 原文

> `{ as: 'url' }` is also supported for loading assets as URLs.

File: guide/features.md [hunk 3]
```diff
@@ -327,7 +327,7 @@ const modules = {
 }
 ```
 
-`{ as: 'url' }` はアセットの URL としての読み込みもサポートしています。
+`{ as: 'url' }` によりアセットの URL としての読み込みもサポートしています。
```
Comment (sapphi-red) [inline guide/features.md:330]: 原文

> `{ as: 'url' }` is also supported for loading assets as URLs.

**Review by jay-es — APPROVED** [review]
ありがとうございます！:pray:

**Pull request merged**